spec/components/discourse_hub_spec.rb

```diff
@@ -9,4 +9,57 @@
       expect(DiscourseHub.discourse_version_check).to eq(hub_response)
     end
   end
+
+  describe '#version_check_payload' do
```

Comment:
You probably wanted `.version_check_payload` since `#version_check_payload` is for instance methods.